### DISCUSSION TITLE
PCF8574

### DISCUSSION THREAD

**Discussion by piepa12** [CrashOverride85/zc95#71] [General]
I have a question. Which PFC8574 did you use? A, AN, At...? 

Greetings Jonas

**Comment by CrashOverride85** [comment]
Hi, I've used the `PCF8574P`

**Comment by piepa12** [comment]
I Have a problem withe the INT2 from U2 (Front Panel). Who the Pico U4 is connected the Interrupt don't go to 0V. Its only changing from 3.3V to 3V. When U4 istdisconnectet the interrupt change between 3.3V and 0V. Did you have a tip for me?

**Comment by piepa12** [comment]
![DSX_20231011202214_2](https://github.com/CrashOverride85/zc95/assets/80581416/f65ccacd-d724-49eb-be96-0533f5ae82dc)

![DSX_20231011202007_2](https://github.com/CrashOverride85/zc95/assets/80581416/d55d05d1-9086-4750-a907-8ab14b3ba57d)

**Comment by CrashOverride85** [comment]
hmmm, that's certainly strange. That pin on the Pico should be an input, but it looks like something is pulling it up rather more strongly than it should be.

Could you check R24 is definitely ~10k, and not something much lower?

**Reply by piepa12** [reply]
R24 ist 10k. I tried with 100k and get the same result.

**Reply by CrashOverride85** [reply]
Ahhh I think I've spotted what's going wrong, no idea why it was working at all for me / others.

Could you try this firmware on the main board, and see if improves things?
[zc95.uf2.zip](https://github.com/CrashOverride85/zc95/files/12896975/zc95.uf2.zip)

**Reply by piepa12** [reply]
It works! Thank you for the fast support. What was the problem? An internal pull-up?

**Reply by CrashOverride85** [reply]
Ahh brilliant, glad it's working :)

No, it was worse than that - I'd mistakenly configured the display driver to use that pin as the back light, so it was driving it high, not just pulling it up. Not sure how it was working at all tbh.

And thanks for letting me know it worked, the fix will be be in v1.7 onwards too
